src/server/server.ts

```diff
@@ -28,7 +28,7 @@ export default class Server implements Hub.RouteBuilder {
       Raven.config(process.env.ACTION_HUB_RAVEN_DSN, {
         captureUnhandledRejections: true,
         release: statusJson.git_commit,
-        autoBreadcrumbs: true,
+        autoBreadcrumbs: false,
```

Comment:
Is this just a precaution?

Reply:
yes, it doesn't provide particularly useful debugging info in our case either